src/api/client/Assets.ts

```diff
@@ -269,31 +269,46 @@ export class Assets {
   /**
    * Retrieve a FungibleAsset
    *
+   * @param args.assetId - Unique Id of the Fungible Asset (for spec version 6.x, this is same as ticker)
    * @param args.ticker - Asset ticker
-   * @param args.assetId - Unique Id of the Asset (for spec version 6.x, this is same as ticker)
+   * @param args.skipExistsCheck - when true, method will not check if the Asset exists
    */
-  public async getFungibleAsset(args: { assetId: string }): Promise<FungibleAsset>;
-  public async getFungibleAsset(args: { ticker: string }): Promise<FungibleAsset>;
+  public async getFungibleAsset(args: {
```

Comment:
Why do we need these extra interfaces? Aren't the covered by the last one?

Reply:
Thinking it twice, I guess the last one is the one which we should remove as the other two ensures there is at least one asset reference

Reply:
Yes, I know the overloaded format. But I mean the third case won't happen where both `ticker` and `assetId` are both not present

Reply:
Last one is not a new variant, its the implementation of the above two formats

Reply:
Haha that's why I should read the code on the IDE rather than here. Beginning of the implementation was hidden